glue/viewers/profile/qt/options_widget.py

```diff
@@ -2,17 +2,14 @@
 
 from qtpy import QtWidgets
 
-from glue.core.coordinate_helpers import dependent_axes
 from echo.qt import autoconnect_callbacks_to_qt
 from glue.utils.qt import load_ui, fix_tab_widget_fontsize
+from glue.viewers.profile.qt.slice_widget import ProfileMultiSliceWidgetHelper
+from glue.viewers.matplotlib.state import MatplotlibDataViewerState
 
 __all__ = ['ProfileOptionsWidget']
 
 
-WARNING_TEXT = ("Warning: the coordinate '{label}' is not aligned with pixel "
-                "grid, so the values shown on the x-axis are approximate.")
```

Comment:
Why remove this?

Reply:
Actually @Cadair has been saying we don't need it if I remember correctly... I could add this back if that is not the case.

Reply:
WCSAxes makes this redundant right? If you plot a coupled axis it now shows both coords.

Reply:
Ah yes of course!

Reply:
Well but wait it is still needed when not in slice mode, since you might be e.g. averaging over many slices with slightly different x axes

Reply:
Okay, let me put it back 👌🏼

Reply:
Yes just checked, and this warning is still needed in non-slice mode

Reply:
I have been playing around with the different settings for a bit... Seems like there is a bug with the warning that would bring back the much dreaded extra sliders even when the function is not `slice`, especially when changing between the `x-axis` options. Let me see if there is a way around it.